CHANGELOG.md

```diff
@@ -12,8 +12,10 @@ To install release candidates run `[sudo] gem install cocoapods --pre`
 
 ##### Bug Fixes
 
-* None.  
-
+* Don't link aggregate targets to a pod's vendored static library when the pod
```

Comment:
I think this should be: "Don't link vendored static libraries to the aggregate targets" and also update your commit message to match that.